downstream/assemblies/platform/assembly-automation-mesh-operator-aap.adoc

```diff
@@ -0,0 +1,33 @@
+[id="assembly-automation-mesh-operator-aap"]
+
+= {AutomationMeshStart} for operator-based {PlatformName}
+
+Scaling your automation mesh is available on Openshift or kubernetes-based deployments of {PlatformName} and is possible through adding or removing nodes from your cluster dynamically, through the *Instances* resource of the {ControllerName} UI, without running the installation script.
```

Comment:
I would remove "kubernetes-based" for now since OpenShift is the only customer-facing k8s model currently supported.